src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -846,6 +846,9 @@ def load_arguments(self, _):
                                                       help="The target location where the Bicep module will be published."))
         c.argument('documentationUri', arg_type=CLIArgumentType(options_list=['--documentationUri', '-d'],
                                                                 help="The documentation uri of the Bicep module."))
+        c.argument('with_source', arg_type=CLIArgumentType(options_list=['--with-source', '--src'],
+                                                           action='store_true',
+                                                           help="[Experimental] Publish source code with the module."))
```

Comment:
```suggestion
        c.argument('with_source', options_list=['--with-source', '--src'], action='store_true', help="Publish source code with the module.", is_preview=True)
```

Reply:
- When you haven't extracted reusable argument type, you don't need to use `CLIArgumentType` to define argument type.
- How about using `is_preview=True` instead of `[Experimental]` label in the help message?